projects/packages/search/src/customizer/class-customizer.php

```diff
@@ -232,6 +232,26 @@ public function customize_register( $wp_customize ) {
 			)
 		);
 
+		$id = $setting_prefix . 'show_post_date';
+		$wp_customize->add_setting(
+			$id,
+			array(
+				'default'              => is_multisite() ? '1' : '0',
```

Comment:
I really appreciate how you preserved current functionality for both single and multisites - as a user, I'd think this was a really nice UX choice!

Reply:
Turns out this was incorrect, the multisite calls in the search refer to multi site search, not multisite the WP setting. Confusing but 🤷 

I'm actually removing this in a follow up PR